dev/view.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <memory>   //  std::shared_ptr
+#include <string>   //  std::string
+#include <utility>  //  std::forward, std::move
+#include <sqlite3.h>
```

Comment:
ITNOA

I think it is good to change order include non standard library from standard library